packages/http-client-csharp/eng/scripts/Generate.ps1

```diff
@@ -16,6 +16,7 @@ if ($null -eq $filter -or $filter -eq "Unbranded-TypeSpec") {
     $testProjectsLocalDir = Join-Path $packageRoot 'generator' 'TestProjects' 'Local'
 
     $unbrandedTypespecTestProject = Join-Path $testProjectsLocalDir "Unbranded-TypeSpec"
+    $unbrandedTypespecTestProject = $unbrandedTypespecTestProject.Replace("\", "/")  # replace \ with / for the path to avoid path-unix-style warning
```

Comment:
There are too many `"\"` to `"/"` conversion, maybe we can use `slash` module: https://www.npmjs.com/package/slash

Reply:
hmm, this is `Powershell` not `js` though.